.github/workflows/integration.yml

```diff
@@ -13,7 +13,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-20.04]
-        python-version: [3.7, 3.8, 3.9]
+        python-version: [3.7, 3.8, 3.9, 3.10]
```

Comment:
don't do `3.10` in yaml as it will be interpreted as number `3.1`, while you want `'3.10'` string

Reply:
:+1:

Reply:
Could you please update your branch?